lib/services/elasticsearch.js

```diff
@@ -883,26 +895,43 @@ class ElasticSearch extends Service {
           errors: []
         };
 
-        let row;
+        let
+          i = 0,
+          row;
+
 
         while ((row = body.items.shift()) !== undefined) {
           const
             action = Object.keys(row)[0],
             item = row[action];
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') "item" hides or potentially hides a variable declared in an outer scope at line 1365. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableShadowing)